src/shogun2-core/shogun2-service/src/main/java/de/terrestris/shogun2/service/UserService.java

```diff
@@ -66,6 +90,24 @@ public User createNewUser(User user, boolean encryptPassword) {
 	}
 
 	/**
+	 *
+	 * @return
+	 */
+	public User updateExistingUser(User user) {
+
+		if(user.getId() == null) {
+			// to be sure that we are in the
+			// "update" case, the id must not be null
+			return user;
+		}
+
+		dao.saveOrUpdate(user);
+
+		return user;
```

Comment:
It is possible that this will even work this way, but it agree that it would be nicer to explicitly return the saved user.